l10n_ro_account_edi_ubl/models/account_edi_xml_cius_ro.py

```diff
@@ -46,6 +46,24 @@ def _get_partner_party_tax_scheme_vals_list(self, partner, role):
                 vals["tax_scheme_id"] = "!= VAT"
         return vals_list
 
+    def _get_tax_category_list(self, invoice, taxes):
+        # EXTENDS account.edi.xml.ubl_21
+        vals_list = super()._get_tax_category_list(invoice, taxes)
+        # for vals in vals_list:
+        #     vals.pop('tax_exemption_reason', None)
+        for vals in vals_list:
+            if "Invers" in taxes.name:
```

Comment:
poate ca trebuie sa adaugam un camp in TVA cu motivele si sa avem mai multe TVA-uri pentru ca asa nu se acopera toate situatiile